index.js

```diff
@@ -1,23 +1,27 @@
-import { Book } from "./modules/addnew";
-import { BookListManger } from "./modules/booklismanager";
-import { DateTime } from "./modules/luxon";
+/* eslint-disable import/extensions */
+/* eslint-disable no-undef */
+/* eslint-disable no-restricted-globals */
```

Comment:
- I see that you have linters disabled, please I suggest that you try fixing the issues raised by linters instead of disabling them. Remember linters helps to ensure that your code is clean and easy to read 👍.

Reply:
@mckent05 
Hello Akinlade,
Thank you for your comment. I have removed every linters disable comment 2 of them. One deals with extensions, it is asking me to remove the .js extension when importing the js file but without adding it, the code doesn't work. The other deals with global restrictions which also can't be removed. Please kindly consider that.